### PR TITLE
Cleaned up Recent posts and made at least one tag mandatory

### PR DESCRIPTION
**Pull request by davidkartuzinski**

### CODE REVIEW

File: blog/models.py [hunk 1]
```diff
@@ -37,23 +37,23 @@ class Meta:
...
 
     @property
     def instagram_profile_url(self):
-        return 'https://instagram.com/' + str(self.instagram)
+        return f'https://instagram.com/{self.instagram}'
 
     @property
     def github_profile_url(self):
-        return 'https://github.com/' + str(self.github)
+        return f'https://github.com/{self.github}'
```
Comment (bbelderbos) [inline blog/models.py:56]: Much nicer

File: blog/templates/blog/related-stories.html [hunk 2]
```diff
@@ -14,32 +14,32 @@ <h3 class="h2">Related Stories</h3>
             <div class="col-md-6 col-lg-4 d-flex" data-aos="fade-up" data-aos-delay="200">
                 <div class="card ">
 
-                                            <a href="{% url 'blog:post_list' %}{{ post.slug }}" class="card-img-top">
-                            <img src="{{ post.header_image.url }}" alt="{{ post.header_image_alt.url }}"
-                                 class="card-img-top">
-                        </a>
+                    <a href="{% url 'blog:post_list' %}{{ post.slug }}" class="card-img-top">
```
Comment (bbelderbos) [inline blog/templates/blog/related-stories.html:17]: Should `post.slug` be part of the `url` rather (e.g. like L35 below)?

File: blog/templates/blog/related-stories.html [hunk 3]
```diff
@@ -50,8 +50,8 @@ <h4>{{ post.title }}</h4>
                 </div>
 
             </div>
-{% empty %}
-    <p class="text-muted">There is no similar post yet</p>
+            {% empty %}
```
Comment (bbelderbos) [inline blog/templates/blog/related-stories.html:53]: The `empty` is neat.

**Review by bbelderbos — COMMENTED** [review]
Looks good

File: blog/templates/blog/related-stories.html (hunk 2 above)
Comment (davidkartuzinski) [reply blog/templates/blog/related-stories.html:17]: I don't quite understand. the `blog:post_list` finds the app and model. and `post.slug` names the data we need. It's not real clear but this is the only way I could get it to work.  

Do you mean more like this, `{% url 'blog:author' author=post.author.get_username %}` that is what is on line 35...

Comment (bbelderbos) [reply blog/templates/blog/related-stories.html:17]: Exactly, that didn't work?

Comment (davidkartuzinski) [reply blog/templates/blog/related-stories.html:17]: @bbelderbos - L35 us working https://ellieplatform.org/blog/author/david -> `{% url 'blog:author' author=post.author.get_username %}`.  I am confused, are you saying L17 should look different?

Comment (bbelderbos) [reply blog/templates/blog/related-stories.html:17]: So this links to a blog post right?

The corresponding route is: `path('<slug:slug>/', views.blog_post, name='blog_post'),`

So I would have expected something like `{% url 'blog_post' slug=post.slug %}`

**Pull request merged**